paper/paper.md

```diff
@@ -0,0 +1,170 @@
+- index: 7
+  name: UC Berkeley, Project Jupyter
+- index: 8
+  name:
+- index: 9
+  name: Department of Biomedical and Clinical Sciences 'L. Sacco', University of Milan, Milan, Italy
+- index: 10
+  name:
+- index: 11
+  name:
+- index: 12
+  name:
+- index: 13
+  name: Institute of Psychology, University of Graz, Austria
+- index: 14
+  name: University of Oregon, Eugene OR, USA
+- index: 15
+  name: Université Paris-Saclay, Inria, CEA, Palaiseau, France
+date: 28 October 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+With the emergence of the Brain Imaging Data Structure
+(``BIDS``; [@Gorgolewski2016]) in 2016, the scientific community received a
+standard to organize and share data in the broad domain of neuroscience.
+
+Originally limited to MRI data types, BIDS is continuously being extended and
+by now also supports other neuroimaging modalities such as MEG [@Niso2018],
+EEG [@Pernet2019], and iEEG [@Holdgraf2019].
+
+BIDS prescribes how complex experiment data should be structured and which
+metadata should be encoded next to the raw data. This set of guidelines
+allows users to reap several benefits, such as:
+
+1. **sharing data within a lab and between labs:** Through reading the BIDS
+   specification, users are empowered to understand any BIDS dataset
+   without requiring specialist knowledge.
+1. **data validation:** Dedicated software can check whether the rules of
+   BIDS are being followed in a dataset, and alert users to missing or broken
+   data (see
+   [bids-validator](https://github.com/bids-standard/bids-validator)).
+1. **automatic data handling and analysis pipelines:** With a perfect
+   knowledge about which data to expect and where to search for it, many
+   common workflows in data handling and analysis can be simplified or
+   automated through specialized software.
+
+To make use of these benefits, datasets must first be converted to
+BIDS format. Furthermore, already existing data analysis software must be
+extended to be aware of a BIDS format and how to properly use it.
+
+This is where ``MNE-BIDS`` steps in: Harnessing BIDS and the data analysis
+software MNE-Python [@Agramfort2013], it is the goal of ``MNE-BIDS`` to
+automatically organize raw MEG, EEG, and iEEG data into BIDS format and to facilitate
+their analysis.
+
+For example, starting with a single directory full of datafiles with arbitrary
+naming, ``MNE-BIDS`` can be used to extract present metadata, reorganize the
+files into BIDS format, and write additional metadata. Moreover,
+``MNE-BIDS`` supports conversion from raw data formats that are not BIDS
+compatible into permissible formats. As a result, users can easily convert
+their datasets to BIDS in a matter of minutes, rather than after hours of
+manual labour. All conversions performed by MNE-BIDS are validated with the
+[bids-validator](https://github.com/bids-standard/bids-validator) for a maximum
+certainty that the results hold up to the BIDS format.
```

Comment:
`or a maximum certainty that the results hold up to the BIDS format.` ->
`to ensure compatibility with the BIDS format.`